quora-service/src/main/java/com/upgrad/quora/service/business/AuthenticationService.java

```diff
@@ -130,4 +131,23 @@ public String checkUserByAuthToken(String authToken, String userId) throws Autho
         userDao.deleteUser(userId);
         return userEntity.getUuid();
     }
+    /**
+     * Get User by Authorization Token.
+     * @param accessToken authToken that is provided during login.
+     * @throws AuthorizationFailedException
+     *
+     */
+    public UserAuthTokenEntity getUserByToken(final String accessToken) throws AuthorizationFailedException {
```

Comment:
this code already exists in one of the function i wrote. Can you sync and make sure that function is also modified